dhis-2/dhis-api/src/main/java/org/hisp/dhis/icon/CustomIconOperationParams.java

```diff
@@ -76,4 +76,8 @@ public boolean hasKeywords() {
   public boolean hasKeys() {
     return !keys.isEmpty();
   }
+
+  public boolean hasCustom() {
```

Comment:
This is an easily misunderstood name. Either find a more communicating name or add a javadoc explaining what this means.

Reply:
fixed